app/config/FrontendAppConfig.scala

```diff
@@ -36,6 +37,7 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   val reportAProblemNonJSUrl = s"$contactHost/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
   val betaFeedbackUrl = s"$contactHost/contact/beta-feedback"
   val betaFeedbackUnauthenticatedUrl = s"$contactHost/contact/beta-feedback-unauthenticated"
+  val agentServiceRegistrationUrl = s"$agentServicesFrontendHost/agent-subscription/start?continue=/trusts-registration/trust-registered-online"
```

Comment:
`/trust-registered-online` This endpoint doesn't exist in the microservice, what should this be?